pgjdbc/src/main/java/org/postgresql/core/v3/plugins/AuthenticationPlugin.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * AWS JDBC Driver for PostgreSQL
+ * Copyright Amazon.com Inc. or affiliates.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.core.v3.plugins;
+
+public interface AuthenticationPlugin {
```

Comment:
Let's add "I" to interface name: IAuthenticationPlugin.

Reply:
Added.